transfers/views/student_views.py

```diff
@@ -67,6 +68,24 @@ def post(self, request, *args, **kwargs):
             'hod_email_list': hod_email_list,
             'invalid_supervisor_email': invalid_supervisor_email
         }
+#Notification System
+        body = ""
+        for data in PS2TSTransfer.objects.all():
+            if request.user == data.applicant.user:
+                body =  str("\nID: " + data.applicant.user.username+
+                    "\nName: " + data.applicant.user.first_name + " " +data.applicant.user.last_name +
+                    "\nTransfer Type: " + data.get_sub_type_display()+
+                    "\nCGPA: " + str(data.cgpa)+
+                    "\nThesis Locale: " + data.get_thesis_locale_display()+
+                    "\nThesis Subject: " + data.thesis_subject+
+                    "\nOrganization Name: " + data.name_of_org+
+                    "\nExpected Outcome: " + data.expected_deliverables)
+                print("done")
+                send_mail("Transfer Application: " + request.user.username, body,
+                    'psdmail@gmail.com',[str(data.hod_email)],
```

Comment:
This email is not the same as one defined in `settings.py`, why is it so?

Reply:
i don't know why it's working this way. This was an unintended error. Also fixed.